connectivity/drivers/802.15.4_RF/CMakeLists.txt

```diff
@@ -1,8 +1,18 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-mbed_add_cmake_directory_if_labels("TARGET")
+if("Freescale" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_Freescale)
+    add_subdirectory(mcr20a-rf-driver)
+elseif("Silicon_Labs" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_Silicon_Labs)
+elseif("STM" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(stm-s2lp-rf-driver)
+endif()
 
 add_subdirectory(atmel-rf-driver)
```

Comment:
Do we need to include `atmel-rf-driver` source even if the target specified with `Freescale` as previous `if` condition for `Freescale` target adding mcr20a-rf-driver

Reply:
I am not sure I understand what you are saying. Can you clarify please? Thanks.